api/namex/constants/__init__.py

```diff
@@ -103,6 +103,12 @@ class EntityTypes(AbstractEnum):
     XPRO_SOCIETY = 'XSO'
     # Used for mapping back to legacy oracle codes
     FIRM = 'FIRM'
+    # Continued In Types
+    CONTINUE_IN = 'C'             # Continued In BC Ltd.
+    CONTINUE_IN_CBEN = 'CBEN'     # Continued In Benefit Company 
```

Comment:
Key should be `CONTINUE_IN_BEN`.

Reply:
@eve-git Please fix this. Thanks.

Reply:
Right now, this says, "a Continued In CBEN is a CBEN". It should say, "a Continued In BEN is a BEN".

In fact, the ULC one is incorrect, too. It should be `CONTINUE_IN_ULC = 'CUL'`.

Reply:
Right now, this says, "a Continued In CBEN is a CBEN". It should say, "a Continued In BEN is a CBEN".

In fact, the ULC one is incorrect, too. It should be `CONTINUE_IN_ULC = 'CUL'`.